docs/en/observability/uptime-set-up-heartbeat.asciidoc

```diff
@@ -0,0 +1,130 @@
+[[uptime-set-up-choose-heartbeat]]
+= Use {heartbeat}
+
+{heartbeat} is a lightweight daemon that you install on a remote server to periodically
+check the status of your services and determine if they are available. Performance data is
+then gathered and reported back to {heartbeat}, where it's formatted and sent to the {stack}.
+
+[discrete]
+[[uptime-set-up-docker]]
+== Pull the Docker image
+
+Elastic provides Docker images that you can use to run monitors, which reduces the chance of problems.
```

Comment:
I'd say it makes it "easier to run", as mentioned previously. Besides not having to advocate too strongly for Docker, IMO saying it "reduces the chance of problems" reflects badly on us (as in: "ah so there are significant chances of problems")